packages/coding-agent/src/modes/interactive/interactive-mode.ts

```diff
@@ -1013,12 +1013,12 @@ export class InteractiveMode {
 				};
 				// Show retry indicator
 				this.statusContainer.clear();
-				const delaySeconds = Math.round(event.delayMs / 1000);
+				const retryTime = new Date(Date.now() + event.delayMs).toLocaleTimeString("en-GB");
 				this.retryLoader = new Loader(
 					this.ui,
 					(spinner) => theme.fg("warning", spinner),
 					(text) => theme.fg("muted", text),
-					`Retrying (${event.attempt}/${event.maxAttempts}) in ${delaySeconds}s... (esc to cancel)`,
+					`Retrying (${event.attempt}/${event.maxAttempts}) at ${retryTime} (esc to cancel)`,
```

Comment:
This is a bit nicer UX, because the server might say `Try again in 18h30m15s`, in which case it is more helpful to see `Retrying at 16:55:01` than `Retrying in ABCXYZs`